### ISSUE TITLE
Java 8 compatability

### ISSUE THREAD

**Issue by patjackson52** [GoogleCloudPlatform/gradle-appengine-plugin#292]
When using Java 8 runtime there is an error for any classes in the old whitelist that stops deployment/local dev.

**Comment by loosebazooka** [comment]
Can you provide more detail?

**Comment by loosebazooka** [comment]
Have you tried the gcloud based plugin https://Github.com/GoogleCloudPlatform/app-gradle-plugin when using java 8?

**Comment by patjackson52** [comment]
When using the Java 8 runtime withis this plugin and previously whitelisted
classes, the plugin can not deploy or run local server.  Appears to be due
to appcfg checking the classes?

Yes, I've tried the other plugin and it does not have a plugin that allows
local dev/debugging within Intellij/Android studio.

**Comment by patflynn** [comment]
@patjackson52 FYI we do have a [plugin for Intellij](https://github.com/GoogleCloudPlatform/google-cloud-intellij).   Unfortunately no support for Android Studio at this time.

**Comment by patjackson52** [comment]
I have found a hack that will work for the being.  If you remove the appengine-agent.jar and appengine-agentimpl.jar from ~/.gradle/appengine-sdk/appengine-java-sdk-{version}/lib/agent/
it skip the class restriction check and run in debug just fine.  Looks like this is out of the gradle plugins control and is under the Intellij plugin.

I've created a gradle tasks that renames the file on assemble tasks so other team members will not have to bother with it.
